src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/Freezer.java

```diff
@@ -17,16 +18,26 @@
 
 public class Freezer extends AContainer implements RecipeDisplayItem {
 
+    private final ItemSetting<Boolean> vanillaTransformation = new ItemSetting<>(this, "vanilla-translation", false);
```

Comment:
I think this setting needs a better name, e.g. `use-vanilla-ice-ratio`.
Although that might make people think of vanilla ice cream XD

Reply:
it depends if more is added in the future... `use-vanilla-ratio` would be better

Reply:
`use-vanilla-ratios` then

Reply:
Do you want me to leave a comment saying dont get confused by ice cream? XD